style.css

```diff
@@ -1,18 +1,652 @@
-/*@import "fonts.css";*/
-
-/*body {
-	font-family: "Montserrat-SemiBold";
-}*/
-
 .header, .start, .specifications, .salon, .description__title, .parameters__item, .safety__title, .stars__subtitle, .fivestars__textstars{
-	font-family: "Montserrat-SemiBold";
+font-family: "Montserrat-SemiBold";
 }
 .specifications__item, .description__link, .stars__item, .fivestars__text, .fivestars__link{
-	font-family: "Montserrat-Bold";
+font-family: "Montserrat-Bold";
 }
 .description__text, .safety__text{
-	font-family: "Montserrat-Medium";
+font-family: "Montserrat-Medium";
 }
 .parameters__small{
-	font-family: "Montserrat-Regular";
+font-family: "Montserrat-Regular";
+}
+
+
+
+
+
+
+
+
+
+body{
+max-width: 1440px;
+border: 1px solid orange;
+margin: 0;
+display: flex;
+flex-direction: column;
+}
+
+@media (max-width:1440px){
+	body{
+	max-width: 1400px;
+	}
+	.header{
+		max-width: 1400px;
+	}
+}
+
+@media (max-width:992px){
+	body{
+	max-width: 768px;
+	}
+	.header{
+		max-width: 768px;
+	}
+}
+
+@media (max-width:767px){
+	body{
+	max-width: 375px;
+	}
+	.header{
+		max-width: 375px;
+	}
+}
+
+.header{
+border: 1px solid orange;
+display: flex;
+max-width: 1440px;
+height: 51px;
+padding-top: 21px;
+justify-content: center;
+align-content: center;
+align-items: center;
+justify-content: space-between;
+}
+
+.header__title{
+	border: 1px solid orange;
+	padding-left: 41px;
+	margin-right: 250px;
+}
+
+.menu{
+border: 1px solid blue;
+font-size: 1vw;
+display: flex;
+justify-content: center;
+align-content: center;
+/*width: max-content;*/
+}
+
+.menu__store{
+	display: flex;
+	justify-content: center;
+	align-content: center;
+	bottom: 20px;
+	border: 1px solid green;
+	margin-left: 160px;
+}
+
+.menu__link{
+	border: 1px solid red;
+	min-width: 65px;
+display: flex;
+align-items: center;
+align-content: center;
+font-weight: 600;
+font-size: 15px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)